Core/include/Acts/Vertexing/HelicalTrackLinearizer.hpp

```diff
@@ -82,9 +82,6 @@ class HelicalTrackLinearizer {
     /// Tolerance determining how close we need to get to the Perigee surface to
     /// reach it during propagation
     ActsScalar targetTolerance = 1e-12;
-
-    // Minimum q/p value
-    double minQoP = 1e-15;
```

Comment:
Breaking?

Reply:
I don't think that this variable was ever not set to its default value; it was just some work-around before `ParticleHypothesis` was introduced.

But let's run the Athena CI to be sure!

Reply:
I propose we merge and let it figure it out for us. I guess technically it is breaking simply because it's a public config a variable, but let's be pragmatic.